MapboxNavigationTests/Support/ImageDownloadOperationSpy.swift

```diff
@@ -60,5 +60,6 @@ class ImageDownloadOperationSpy: Operation, ImageDownload {
         completionBlocks.forEach { completion in
             completion(image, data, error)
         }
+        RunLoop.current.run(until: Date().addingTimeInterval(5))
```

Comment:
Oof. this is punitive. can we reduce this, with the goal of eliminating it? I thought one tick of the runloop was enough here. was that not the case on your CPU-bound machine?

Reply:
I pushed an update to adjust this, since things passed for me while under a sustained load average of ~80+